tests/lib/cli-engine.js

```diff
@@ -923,8 +923,51 @@ describe("CLIEngine", function() {
                 assert.equal(report.results.length, 1);
                 assert.equal(report.results[0].messages.length, 2);
             });
-        });
+            it("should run the preprocessor when executing with config file that specifies a processor", function() {
```

Comment:
This test also tests the functionallity of the `postprocessor`, so the title is a bit inaccurate.